spec/fountain.spec.ts

```diff
@@ -287,6 +287,17 @@ describe('Inline markdown lexer', () => {
         let actual = output.html.script;
         let expected = '<p><span class=\"bold italic underline\">bold italics underline</span> <span class=\"bold underline\">bold underline</span> <span class=\"italic underline\">italic underline</span> <span class=\"bold italic\">bold italics</span> <span class=\"bold\">bold</span> <span class=\"italic\">italics</span> <span class=\"underline\">underline</span></p>';
 
-        expect(expected).toBe(actual);
+        expect(actual).toEqual(expected);
+    });
+
+    it('should use OOP token objects that are backwards-compatible with old versions', () => {
+        const text = 'The SCREEN DOOR slides ...es with two cold beers.';
+        const legacyToken = { type: 'action', text };
+        const oopToken = new ActionToken(text);
+
+        expect(oopToken).toBeInstanceOf(Object);
+        expect(JSON.stringify(oopToken)).toEqual(JSON.stringify(legacyToken));
+        expect(oopToken.text).toEqual(legacyToken.text);
+        expect(oopToken.type).toEqual(legacyToken.type);
```

Comment:
This confirms that the OOP tokens are backward-compatible with old versions. Definitely feel free to tack on other assertions if you are so inclined!